dev-haskell/ghc-api-compat/ghc-api-compat-8.6.1_pre20210712.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+# The version on hackage isn't properly packaged, so we use the live version
+# for now
+inherit haskell-cabal git-r3
+
+DESCRIPTION="GHC-API compatibility helpers"
+HOMEPAGE="https://hackage.haskell.org/package/ghc-api-compat"
+#SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+EGIT_REPO_URI="https://gitlab.haskell.org/haskell/ghc-api-compat.git"
+EGIT_COMMIT="1c4b23e8f929b052f2172ca509c7428811547b89"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
```

Comment:
The `ghc-api-compat` package doesn't come with a `src` dir so it fails immediately. I thought grabbing the version from github would be a reasonable solution, but now pkgcheck is complaining about it having the `~amd64` keyword. I could either:

- Switch to `KEYWORDS=""`
- Include a properly packaged tarball, but where would I put it?

Reply:
Since the compressed tree for this package is only 6155 bytes, I think it's reasonable to properly package it and stick the tarball in `$FILESDIR`.